cordova-plugin-outline/apple/vpn/PacketTunnelProvider.m

```diff
@@ -271,9 +271,8 @@ - (void)connectTunnel:(NEPacketTunnelNetworkSettings *)settings
 }
 
 - (NEPacketTunnelNetworkSettings *) getTunnelNetworkSettings {
-  // TODO(alalama): check if the address is free, choose from pool.
-  NEIPv4Settings *ipv4Settings = [[NEIPv4Settings alloc] initWithAddresses:@[@"192.168.20.2"]
-                                                               subnetMasks:@[@"255.255.255.0"]];
+  NEIPv4Settings *ipv4Settings = [[NEIPv4Settings alloc] initWithAddresses:@[ @"192.168.20.1" ]
```

Comment:
Why are we using this address? That was kind of the concern in the first place.

Reply:
We must assign a routable address to the VPN so that the TUN interface virtually connects to it. localhost does not work. Do you have an alternative?

Reply:
What happens if that address is already taken?